instrumentation/kafka/kafka-clients/kafka-clients-0.11/metadata.yaml

```diff
@@ -7,3 +10,11 @@ configurations:
     description: Enables the capture of the experimental consumer attribute "kafka.record.queue_time_ms"
     type: boolean
     default: false
+  - name: otel.instrumentation.messaging.experimental.capture-headers
+    description: Allows configuring headers to capture as span attributes.
```

Comment:
Could start with `A comma-separated list of header names.` like the description of `otel.instrumentation.http.client.capture-request-headers` in https://opentelemetry.io/docs/zero-code/java/agent/instrumentation/http/